py-polars/tests/unit/interchange/test_from_dataframe.py

```diff
@@ -176,11 +176,9 @@ def test_from_dataframe_pandas_boolean_bytes() -> None:
     expected = pl.Series("a", [True, False]).to_frame()
     assert_frame_equal(result, expected)
 
-    with pytest.raises(
-        CopyNotAllowedError,
-        match="byte-packed boolean buffer must be converted to bit-packed boolean",
-    ):
-        result = pl.from_dataframe(df, allow_copy=False)
```

Comment:
this no longer raises with `allow_copy=False` - I don't know how strict (if at all?) the pycapsule interface is about copies, will take a look

Reply:
I checked and none of the examples in https://github.com/search?q=%22pl.from_dataframe%22&type=code use `allow_copy=False`, so I'm not sure how much we should be concerned?

I'd like to suggest just deprecating the argument tbh